### PR TITLE
Add a warning about re-flashing the boot image after an Ubuntu update

### PR DESCRIPTION
**Pull request by Flaburgan**

### CODE REVIEW

File: userguide/dailyuse/anbox.rst [hunk 1]
```diff
@@ -25,6 +25,9 @@ How to install
 
 .. warning::
     Installing Anbox is only recommended for experienced users.
+    
+.. warning::
+    To install Anbox, you need to flash a specific boot image. **You will need to re-flash that image after each Ubuntu Touch update** so when you update from OTA-10 to OTA-11 for example. Not doing so could lead to loop crashes of the Network Manager for example, putting your CPU to 100% and killing your battery in minutes. Always update your Ubuntu Touch only when you have a computer with you to re-flash that boot image.
```
Comment (doniks) [inline userguide/dailyuse/anbox.rst]: are the loop crashes of nm a usual problem with anbox and updates? if so, do you have a reference (not necessarily to be added to the docs, just for my understanding). if not, then I think it would be better to either drop the "Not doing so" sentence or rephrase it to a more generic warning.

Comment (Flaburgan) [reply userguide/dailyuse/anbox.rst]: See [this issue](https://github.com/ubports/ubuntu-touch/issues/1222) I encountered when not flashing the Anbox boot image. I als hesitated to be that precise. I honestly don't know, up to you.

Comment (doniks) [reply userguide/dailyuse/anbox.rst]: I see. yeah, please remove the sentence or make it more generic

Comment (Flaburgan) [reply userguide/dailyuse/anbox.rst]: "Not doing so could put your phone on an unstable state"?

Comment (doniks) [reply userguide/dailyuse/anbox.rst]: mhm. ok. how about "Not doing so can put your phone into an unstable state." ?

Comment (Flaburgan) [reply userguide/dailyuse/anbox.rst]: Sure.

Comment (Flaburgan) [reply userguide/dailyuse/anbox.rst]: PR updated.
